components/transaction/internal/ports/http/routes.go

```diff
@@ -4,13 +4,14 @@ import (
 	"github.com/LerianStudio/midaz/common/mcasdoor"
 	"github.com/LerianStudio/midaz/common/mlog"
 	lib "github.com/LerianStudio/midaz/common/net/http"
+	a "github.com/LerianStudio/midaz/components/transaction/internal/domain/assetrate"
```

Comment:
please, use **ar** for **asset_rate** because we are using **a** for **accounts**, just to normaliza the imports.